ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -443,104 +443,170 @@ def test_ip_access_to_server(self):
 class BaremetalIdracWSManDeployNode(bsm.BaremetalStandaloneScenarioTest):
 
     mandatory_attr = ['driver']
-
     api_microversion = '1.31'  # to set the deploy_interface
     driver = 'idrac'
     image_ref = CONF.baremetal.whole_disk_image_ref
     boot_interface = 'ipxe'
 
-    @decorators.idempotent_id('c2bebda2-fd27-4b10-9015-f7d877f0eb60')
+    @classmethod
+    def skip_checks(cls):
+        super(BaremetalIdracWSManDeployNode, cls).skip_checks()
+        if not CONF.baremetal_feature_enabled.ipxe_enabled:
+            skip_msg = ("Skipping this TC since ipxe is disabled")
+            raise cls.skipException(skip_msg)
+
     @utils.services('image', 'network')
-    def test_ip_access_to_server(self):
+    @decorators.idempotent_id('b467889b-aba4-4696-9e6f-4ef60a3f78a4')
+    def test_deploy_node(self):
         self.boot_and_verify_node()
 
 
-class BaremetalIdracWSManDeployNodeipxe(BaremetalIdracWSManDeployNode):
+class BaremetalIdracWSManDeployNodepxe(bsm.BaremetalStandaloneScenarioTest):
+
+    mandatory_attr = ['driver']
+    api_microversion = '1.31'  # to set the deploy_interface
+    driver = 'idrac'
+    image_ref = CONF.baremetal.whole_disk_image_ref
+    boot_interface = 'pxe'
 
     @classmethod
     def skip_checks(cls):
-        super(BaremetalIdracWSManDeployNodeipxe, cls).skip_checks()
+        super(BaremetalIdracWSManDeployNodepxe, cls).skip_checks()
         if CONF.baremetal_feature_enabled.ipxe_enabled:
-            skip_msg = ("Test covered with boot interface as ipxe")
+            skip_msg = ("Skipping the Test case since ipxe is enabled")
```

Comment:
Skipping the test case.... : t in lower case

Reply:
Done